### ISSUE TITLE
Network problem

### ISSUE THREAD

**Issue by PotatoLearning** [derv82/wifite2#316]
Hi, when I try to run wifite I have some problems like "Conflicting processes: NetworkManager (PID 20557)" and "airmon-ng did not find any wireless interfaces". How could I fix this? 
Thanks a lot

**Comment by Rishabh-Kumar-07** [comment]
Hey, It appears your network manager process is conflicting with Wifite. It would be more helpful if you could provide your os details.

**Comment by PotatoLearning** [comment]
Hi! I'm working with a virtual machine (VMware Workstation 16 player) on windows 10.  On the virtual machine settings I bridged my network connection and I've allowed all my network adapters (Intel(R) dual band wireless-AC 3160, killer E2200 gigabit ethernet controller, microsoft wi-fi direct virtual adapter #3, microsoft wi-fi direct virtual adapter #4 ). 
And on my panel on kali there's no wifi options and It looks like I'm connected through wired conection (Ethernet network connection (Wired connection1 active)).
Thanks for your help
